src/index.tsx

```diff
@@ -1,19 +1,127 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+
+const Counter = (props) => {
+	const [value, setValue] = React.useState(props.initialValue);
+
+	function increment() {
+		setValue(value + 1);
+	}
+
+	function decrement() {
+		setValue(value - 1);
+	}
+
+	return React.createElement(
```

Comment:
there is no any issue here, but it's a bit cleaner to simply use distructuring here as well. 

`import { createElement } from 'react';`

`return createElement(`